.github/auto-merge.yml

```diff
@@ -1,3 +1,3 @@
 - match:
     dependency_type: all
-    update_type: 'semver:major'
+    update_type: 'semver:minor'
```

Comment:
i guess patch also

Reply:
some nice ideas here, similar to our other auto-merge.yml files https://github.com/marketplace/actions/enable-github-automerge

Reply:
Oh I'd expect `minor` to also include what is smaller that minor (so to include `patch` in it). Isn't it?

Reply:
Oh it does include `patch`
![image](https://user-images.githubusercontent.com/1090716/196454324-231dc7e7-a5e6-4144-b127-3202293a6695.png)
(cf. https://github.com/ahmadnassri/action-dependabot-auto-merge#configuration-file-syntax)